Framework/Core/include/Framework/LogParsingHelpers.h

```diff
@@ -34,6 +35,9 @@ struct LogParsingHelpers {
   /// Available log levels
   static char const* const LOG_LEVELS[(int)LogLevel::Size];
 
+  /// Map from ROOT log string to log level
+  static const std::map<std::string, LogLevel> MAP_ROOT_LOG_LEVELS;
```

Comment:
std::map is really for sorted lookups where you keep adding and removing from the map. Maybe you should use a std::unordered_map. In any case, please move it out of the header.

Reply:
removed this map, used an if...else block instead like for the Fair logs.